astroid/manager.py

```diff
@@ -59,6 +61,16 @@ class AstroidManager:
     max_inferable_values: ClassVar[int] = 100
 
     def __init__(self):
+        # pylint: disable=import-outside-toplevel
```

Comment:
Are we sure this is unavoidable? This is likely to generate many cyclic imports headaches at some point..

Reply:
I moved the imports even later, inside `clear_cache()`. But yes, this is because a cyclic import is unavoidable--those modules import manager.